ipfsApi/utils.py

```diff
@@ -3,143 +3,12 @@
 
 from __future__ import absolute_import
 
-import io
-import json
 import mimetypes
 import os
 from functools import wraps
 
 import six
 
-from six.moves import cPickle as pickle
```

Comment:
Yes, it was mostly because I've been doing so much refactoring in the last couple days, I just batched the PRs by time of creation. Originally exception handling and unicode handling where even stuffed into one big commit, but I split it up later; probably at least should have done the same thing for the `pickle` en- and decoder. :expressionless: 

I'll try to better myself in the future. :blush: